Tests/InstantSearchCoreTests/Unit/SequencerTest.swift

```diff
@@ -179,39 +177,23 @@ class SequencerTest: XCTestCase {
     let testQueue = OperationQueue()
     testQueue.maxConcurrentOperationCount = 200
 
+    // Launch 100 slow operations and 1 fast operation and add them to the sequencer supporting 101 concurrent operations
     slowOperations.forEach { operation in
       testQueue.addOperation(operation)
       sequencer.orderOperation { operation }
     }
     testQueue.addOperation(fastOperation)
     sequencer.orderOperation { fastOperation }
 
+    // Wait for success of the fast operation
     waitForExpectations(timeout: 100, handler: .none)
 
+    // All slow operations must be cancelled as the seqNo of the fast operation is bigger than the seqNo of the slow operations
     XCTAssertEqual(slowOperations.filter { $0.isCancelled }.count, slowOperationsCount)
+    
+    // Fast operation shouldn't be cancelled
     XCTAssertFalse(fastOperation.isCancelled)
 
   }
-
-  func testPendingOperations() {
```

Comment:
So I see we removed a test here. Is there a new test that we can write that would fail with the old sequencer code, but succeed with this new one (in a way, that reproduces the crash due to unsynchronised sequencer)

Reply:
I removed the `hasPendingOperations: Bool` property and it test as it was used nowhere, but it's no problem to restore it.
The crash of the unsynchronized sequencer is `EXC_BAD_ACCESS`, so it happens at a random moment while accessing  `pendingOperations` property of the Sequencer with the existing tests while testing with Xcode 12.
I switched the Bitrise environment to Xcode 12, so the fact that the Bitrise check has passed shows that this crash doesn't happen anymore.